### DISCUSSION TITLE
How to set orderingTerm

### DISCUSSION THREAD

**Discussion by arefrafei92** [simolus3/drift#2272] [Q&A] [answered]
I was using this code but after updating it doesn't work.
```
Stream<List<Category>> categoriesList() {
    final q = (select(categories)..where((tbl) => tbl.parent.equals(0)));
    q..orderBy([(tbl) => OrderingTerm.desc(tbl.position)]);

    return q.watch();
  }
```

`The argument type 'GeneratedColumn<int?>' can't be assigned to the parameter type 'Expression<Object>'.`

and also this code :
 ```
void updateReserveNotify(String id) {
    update(notifications)
      ..where((tbl) => tbl.itemIds.contains(id))
      ..write(NotificationsCompanion(view: Value(true)));
  }
```
`The method 'contains' isn't defined for the type 'GeneratedColumn'.`

how can I fix them ? there isn't any update in the documentation.

**Comment by simolus3** [comment, marked as answer]
Can you check that the generated outputs are up-to-date? E.g. run `flutter pub run build_runner build --delete-conflicting-outputs` again after upgrading `drift` and `drift_dev`. Can you also report which version of `drift` and `drift_dev` you're using now?

**Reply by arefrafei92** [reply]
```
drift: ^2.4.2
drift_dev: ^2.4.1
```
I used this line `flutter pub run build_runner build --delete-conflicting-outputs` and now it works, thanks
